src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Added access decision strategy to override access decisions by voter service priority
  * Added `IS_ANONYMOUS`, `IS_REMEMBERED`, `IS_IMPERSONATOR`
  * Hash the persistent RememberMe token value in database.
+ * Bypassed session usage reporting for statefull firewalls
```

Comment:
I'm not sure what this means. Is there really something to add in the changelog? I'm not sure :)

Reply:
Indeed ^^